### PR TITLE
chore: upgrade dependencies

### PR DESCRIPTION
**Pull request by lifeiscontent**
- upgrade dev dependencies
- make main build command faster by parallelizing the work across formats

## Types of changes

- [x] Updated docs / Refactor code / Added a tests case (non-breaking change)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I have updated the changelog.

### CODE REVIEW

**Comment by lifeiscontent** [comment]
side notes:

- I noticed there's still a .travis.yml in the repo, I can delete it if you guys want.
- should probably add Node 16 to GitHub Action as it's the LTS version

**Comment by lifeiscontent** [comment]
CC: @robertrico

**Comment by lifeiscontent** [comment]
CC: @PodaruDragos

**Comment by PodaruDragos** [comment]
@lifeiscontent LGTM, thanks.

@jakehamtexas what do you think ?

Edit: re-running this again has failed. there are still some problems in the CI

**Comment by dcavanagh** [comment]
Maybe revert Sinon?

File: package.json [hunk 1]
```diff
@@ -44,32 +44,33 @@
...
     "karma-sinon": "1.0.5",
-    "mocha": "9.1.2",
+    "mocha": "9.2.2",
+    "npm-run-all": "4.1.5",
     "nyc": "15.1.0",
     "publish-please": "5.5.2",
     "reflect-metadata": "0.1.13",
-    "sinon": "9.2.4",
-    "ts-node": "10.3.0",
+    "sinon": "13.0.2",
```
Comment (dcavanagh) [inline package.json]:
```suggestion
    "sinon": "9.2.4",
```

**Comment by lifeiscontent** [comment]
@dcavanagh didn't seem to do it, any other ideas?

**Comment by notaphplover** [comment]
Such a shame we never merged this. Fortunately dependencies are sort of updated these days. Closing this PR.

**Pull request closed without merging**